addon/tailwind/components/bourbon-modal.css

```diff
@@ -90,86 +90,86 @@
   padding: 0;
 }
 
-.bourbon-modal__close-button:focus {
+.BourbonModal-closeButton:focus {
   outline: none;
 }
 
-.bourbon-modal__close-button svg {
+.BourbonModal-closeButton svg {
   @apply .bourbon-text-grey-light;
   @apply .bourbon-stroke-current;
 }
 
-.bourbon-modal__close-button:hover {
+.BourbonModal-closeButton:hover {
   @apply .bourbon-cursor-pointer;
 }
 
-.bourbon-modal__footer {
+.BourbonModal-footer {
   display: flex;
   justify-content: space-between;
   align-items: center;
 
   @apply .bourbon-mt-6;
 }
 
-.bourbon-modal__link {
+.BourbonModal-link {
   @apply .bourbon-text-sm;
   @apply .bourbon-no-underline;
   @apply .bourbon-text-slate;
   @apply .bourbon-w-full;
 }
 
-.bourbon-modal__button-wrapper {
+.BourbonModal-buttonWrapper {
   display: flex;
   justify-content: flex-end;
   @apply .bourbon-w-full;
 }
 
-.bourbon-modal__content-area {
+.BourbonModal-contentArea {
   padding: 20px 0;
 }
 
 /* scrollable modal styles */
-.bourbon-modal--scrollable {
+.BourbonModal--scrollable {
   padding: 35px 0;
   display: grid;
   grid-template-rows: 60px auto 60px;
   height: 600px;
 }
 
-.bourbon-modal--scrollable .bourbon-modal__close-button {
+.BourbonModal--scrollable .BourbonModal-closeButton {
   right: 8px;
 }
 
-.bourbon-modal--scrollable .bourbon-modal__content {
+.BourbonModal--scrollable .BourbonModal-content {
   overflow-y: auto;
   padding: 0 35px;
   margin-bottom: 0;
 }
 
-.bourbon-modal__hide-footer .bourbon-modal__content {
+.BourbonModal--hideFooter .BourbonModal-content {
   margin-bottom: 0;
 }
 
-.bourbon-modal__hide-footer .bourbon-modal__footer {
+.BourbonModal--hideFooter .BourbonModal-footer {
   display: none;
 }
 
-.bourbon-modal--scrollable .bourbon-modal__footer {
+.BourbonModal--scrollable .BourbonModal-footer {
   @apply .bourbon-mt-1;
   @apply .bourbon-pt-8;
   box-shadow: 0px -3px 6px -2px rgba(0, 0, 0, .1);
   padding-right: 20px;
   padding-left: 30px;
 }
 
-.bourbon-modal__no-padding {
+.BourbonModal_noPadding{
```

Comment:
This rule (and the next rule) should probably be `.BourbonModal--noPadding` right?

Reply:
yep!  good catch!  thanks. :)